functoria-runtime.opam

```diff
@@ -13,7 +13,6 @@ license:      "ISC"
 tags:         ["org:mirage"]
 
 build: [
-  ["dune" "subst"] {pinned}
```

Comment:
Uhm, I'm not in favour of this change. The execution of `dune subst` in a development build helps to track down version numbers (otherwise `%%VERSION%%` is reported). Can we please keep that as is? What is the reasoning behind that change?

The line should be (otherwise some linter complain):
`["dune" "subst"] {dev}`

Reply:
For sure :+1:.

Reply:
for the release in opam-repository, this does not matter (since the watermarks are substituted during the release process anyways) -- but for our main branch I find the `dune subst` really helpful. Thanks a lot :)